project_euler/problem_67/sol1.py

```diff
@@ -0,0 +1,37 @@
+'''
+Problem Statement:
+By starting at the top of the triangle below and moving to adjacent numbers on
+the row below, the maximum total from top to bottom is 23.
+3
+7 4
+2 4 6
+8 5 9 3
+That is, 3 + 7 + 4 + 9 = 23.
+Find the maximum total from top to bottom in triangle.txt (right click and
+'Save Link/Target As...'), a 15K text file containing a triangle with
+one-hundred rows.
+'''
+from __future__ import print_function
+try:
+    raw_input          # Python 2
+    xrange
+except NameError:
```

Comment:
This repo does not support legacy Python.  Please remove __print_function__, __raw_input__, and __xrange__.

Reply:
Removed in the new proposal.